### ISSUE TITLE
Wave Packet screen - control panel overlaps ResetAllButton on Chromebook

### ISSUE THREAD

**Issue by arouinfar** [phetsims/fourier-making-waves#161]
The control panel on the Wave Packet screen overlaps with the ResetAllButton on Chromebook, so we'll need to tighten things up a bit.
<img src="https://user-images.githubusercontent.com/8419308/130641103-5a14381e-79fb-4dce-a958-3d033672bbd5.png" width=200>

@pixelzoom here are some things we can try to save space:

* Reduce the size of the component spacing tool icons
* Shift ResetAllButton down a few px

**Comment by pixelzoom** [comment]
This is undoubtedly due to differences in font size.

Reduce the size of the component spacing tool icons is a good thought. But if I make them smaller, you won't be able to see the label.  @arouinfar Can I omit the label from those icons? We don't include the icon on the width indicator icon. 

I'm also going to add `maxHeight` to the control panel, for a bit of extra protection. In the event that the control panel does exceed the available vertical space, the control panel will be uniformly scaled down to fit.

**Comment by arouinfar** [comment]
Yes, let's go ahead and lose the tool icon labels. We often use iconic representations for tools as checkbox labels and I think it's a reasonable thing to do here too.

**Comment by pixelzoom** [comment]
Hmm... I didn't think I'd miss the labels, but I do.  Now I'm left wondering (or trying to remember) which tool is which. Maybe that's OK.   Or we could put the labels to the left of the icons?

@arouinfar thoughts?

<img width="286" alt="screenshot_1214" src="https://user-images.githubusercontent.com/3046552/130670498-23b8c11f-26e4-44d9-8854-c6ccf46d95dd.png">


<img width="295" alt="screenshot_1213" src="https://user-images.githubusercontent.com/3046552/130670392-a961960a-0125-4534-abe9-80ac32d94429.png">

**Comment by pixelzoom** [comment]
Labels to the left of icons looks best to me, to I went with that. It doesn't both me that the label is in a different location than it is on the interactive tool.  And as a bonus, we can make the labels a bit larger.  Screenshot below.   

I also set `maxWidth` and `maxHeight` for the control panel, as a fallback. In case we miss something, this will guarantee that the control panel doesn't overlap anything.  Tested with `?stringTest`.

@arouinfar please review in master.

<img width="270" alt="screenshot_1216" src="https://user-images.githubusercontent.com/3046552/130671954-bf965e1a-7fcb-426d-962a-ab908f9cbd7d.png">
